.ci/dist-tag.groovy

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'linux && immutable' }
+  environment {
+    REPO = "synthetics"
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    NPMRC_SECRET = 'secret/jenkins-ci/npmjs/elasticmachine'
+    TOTP_SECRET = 'totp/code/npmjs-elasticmachine'
+    NPM_PACKAGE = '@elastic/synthetics'
+    SLACK_CHANNEL = '#synthetics-user_experience-uptime'
+  }
+  options {
+    timeout(time: 1, unit: 'HOURS')  // to support releases then we will add a timeout in each stage
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '20', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    rateLimitBuilds(throttle: [count: 60, durationName: 'hour', userBoost: true])
+    quietPeriod(10)
+  }
+  parameters {
+    string(name: 'VERSION', defaultValue: '', description: 'Version to dist tag.')
+    string(name: 'DIST_TAG', defaultValue: '', description: 'Dist tag to use.')
+  }
+  stages {
+    stage('Checkout') {
+      options {
+        skipDefaultCheckout()
+        timeout(5)
+      }
+      steps {
+        deleteDir()
+        gitCheckout(basedir: "${BASE_DIR}")
+      }
+    }
+    stage('Dist tag') {
+      options {
+        skipDefaultCheckout()
+      }
+      steps {
+        withNodeJSEnv() {
```

Comment:
Minor validation in case the parameters are empty

```suggestion
       whenFalse(env.VERSION?.trim()) {
         error("VERSION cannot be empty: ${env.VERSION}")
       }
       whenFalse(env.DIST_TAG?.trim()) {
         error("DIST_TAG cannot empty: ${env.DIST_TAG}")
       }
        withNodeJSEnv() {
```

Reply:
I added minor validation in a separate stage a bit earlier to avoid unneeded git checkout and so on.